.github/pull_request_template.md

```diff
@@ -8,6 +8,8 @@ Fixes #SOME_ISSUE_NUMBER
 
 Changes proposed in this pull request:
 
-- 
-- 
-- 
+-
+-
+-
+
+When you're ready to submit your PR, dont forget to add the `presubmit` label.
```

Comment:
dont => dont'

It sounds like what the presubmit action will do is then immediately _remove_ the presubmit label and just use it as a signal to start the process. Is that right?

Can we change the label to start with e.g., "#" or "+" so that it sorts before other labels?

Reply:
typo to fix still